### DISCUSSION TITLE
Hi I'm a web dev and i'd like to start building some watch faces

### DISCUSSION THREAD

**Discussion by kenguie** [zepp-health/zeppos-forum#144] [Developer Feedback]
Hi there, I was hoping to develop some watch faces for myself that have everything on it if i can fit it :D I'm on a data team so my watch is the same, i want to see everything the way i want :D 

Where can i get some good tips on how to get resources and build some that resemble the great ones that are on the app?
For example, i did take the example for the GTR3 Pro, edited it with some better looking numbers and resize it for the GTR4 i have but where to i get more numbers? graphics for steps, power, etc? Is there a repository with a common collection or is everyone making their own?

Also, How can i contact zepp health. I love their Function face but maybe forever, the step count at the bottom that should be between the graphic and sun rise sun set is not there! Can someone fix it? It's one of my fav faces. 

Thanks!

**Comment by MyoungXUE** [comment]
Hello, I'm the official Zepp Health staff member and welcome you to the Zepp OS technical community!

Zepp OS watchface offer a rich display of data, try our online Watchface Maker https://docs.zepp.com/docs/guides/tools/watchface/.

We also have official examples examples, please refer to https://docs.zepp.com/docs/samples/

As for your feedback on the Function watchface, I will feedback to the relevant colleagues to solve the problem.

Finally, have fun at Zepp OS!

**Reply by kenguie** [reply]
Thanks for the warm welcome. I've noticed that the watch numbers can't be resized. I guess I need to size them before i use them.
By resources in my question above, is there a good place to get number styles? some already made grids for health info for backgrounds, meters for steps and pai, or anything like that? Kind of like Font Awesome? https://fontawesome.com/icons 

Thanks!

**Comment by MyoungXUE** [comment]
Are you referring to a watchface called Function, and can you provide a screenshot of the watchface to help us pinpoint the problem?

**Reply by kenguie** [reply]
![2023-04-19 10 11 31](https://user-images.githubusercontent.com/7780845/233102651-a266f9c8-70a8-475c-b41f-7b4bf6ef380b.jpg)

**Reply by kenguie** [reply]
Here's your stock photo: 
![2023-04-19 10 14 55](https://user-images.githubusercontent.com/7780845/233103296-f95b3c48-56e2-4584-ac1b-63e8cb637ed7.jpg)

**Comment by kenguie** [comment]
Hi, i was wondering if there was any update to answers to my questions, thanks!

**Reply by MyoungXUE** [reply]
Sorry I forgot to reply to you, I am contacting relevant colleagues to deal with, please wait patiently, I will inform you of the progress.

**Reply by MyoungXUE** [reply]
@kenguie Hi, we have released a fixed version of the watchface, please update it in the app store.

**Reply by kenguie** [reply]
Hi Myoung, I did see that and it's great, thanks!
